src/main/kotlin/pt/up/fe/ni/website/backend/service/ProjectService.kt

```diff
@@ -22,6 +29,11 @@ class ProjectService(private val repository: ProjectRepository) {
     fun updateProjectById(id: Long, dto: ProjectDto): Project {
         val project = getProjectById(id)
         val newProject = dto.update(project)
+        newProject.teamMembers.clear()
+        for (idAccount in dto.teamMembersIds.orEmpty()) {
+            val account = accountService.getAccountById(idAccount)
+            newProject.teamMembers.add(account)
+        }
```

Comment:
Same comment with forEach. You can also try exploring the `apply { }` function to combine clear and forEach if you'd like